### PR TITLE
4 new meme templates

### PR DESCRIPTION
**Pull request by pedall**
- disaster girl
- confused gandalf
- mocking spongebob
- unpopular opinion puffin

### CODE REVIEW

File: data/templates/gandalf/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Confused Gandalf
+link: http://knowyourmeme.com/memes/confused-gandalf
+default:
+- ''
+- What did I just say?
+aliases:
+- gandalf
```
Comment (jacebrowning) [inline data/templates/gandalf/config.yml]: This line should. be removed as it's already the default key for this template based on the directory name.

**Comment by pedall** [comment]
@jacebrowning removed the line as requested

**Comment by jacebrowning** [comment]
I think Travis CI might just be caching an older failure. I'll fix this on `master` if there are any issues. Thanks!

**Comment by pedall** [comment]
@jacebrowning what is the process until this becomes available on the API? ;-)

**Pull request merged**